samples/highcharts/website/xmas-2021/demo.js

```diff
@@ -0,0 +1,3937 @@
+        },
+        {
+            name: 'Barbados',
+            value: 2326
+        },
+        {
+            name: 'Bermuda',
+            value: 18657
+        },
+        {
+            name: 'British Indian Ocean Territory',
+            value: 16667
+        },
+        {
+            name: 'Brunei',
+            value: 190
+        },
+        {
+            name: 'Faroe Islands',
+            value: 718
+        },
+        {
+            name: 'Gambia',
+            value: 99
+        },
+        {
+            name: 'Gibraltar',
+            value: 153846
+        },
+        {
+            name: 'Jan Mayen',
+            value: 2653
+        },
+        {
+            name: 'Jersey',
+            value: 8621
+        },
+        {
+            name: 'Macau',
+            value: 35461
+        },
+        {
+            name: 'Malta',
+            value: 3165
+        },
+        {
+            name: 'Isle of Man',
+            value: 1748
+        },
+        {
+            name: 'Montserrat',
+            value: 9804
+        },
+        {
+            name: 'Nauru',
+            value: 47170
+        },
+        {
+            name: 'Niue',
+            value: 3846
+        },
+        {
+            name: 'Paracel Islands',
+            value: 129032
+        },
+        {
+            name: 'Saint Barthelemy',
+            value: 40000
+        },
+        {
+            name: 'Saint Helena, Ascension and Tristan da Cunha',
+            value: 2538
+        },
+        {
+            name: 'Saint Martin',
+            value: 18382
+        },
+        {
+            name: 'Sint Maarten',
+            value: 29412
+        },
+        {
+            name: 'Tuvalu',
+            value: 39063
+        },
+        {
+            name: 'Wake Island',
+            value: 153846
+        }
+    ];
+    let chart;
+    const getGraticule = () => {
+        const data = [];
+
+        // Meridians
+        for (let x = -180; x <= 180; x += 15) {
+            data.push({
+                geometry: {
+                    type: 'LineString',
+                    coordinates: x % 90 === 0 ? [
+                        [x, -90],
+                        [x, 0],
+                        [x, 90]
+                    ] : [
+                        [x, -80],
+                        [x, 80]
+                    ]
+                },
+                color: '#fff',
+                lineWidth: 0
+            });
+        }
+
+        // Latitudes
+        for (let y = -90; y <= 90; y += 10) {
+            const coordinates = [];
+            for (let x = -180; x <= 180; x += 5) {
+                coordinates.push([x, y]);
+            }
+            data.push({
+                geometry: {
+                    type: 'Polygon',
+                    coordinates
+                },
+                lineWidth: y === 1 ? 0 : undefined
+            });
+        }
+
+        return data;
+    };
+
+    // Add flight route after initial animation
```

Comment:
Is this flight route used? I can't see it in the globe view. I should probably be removed (including its call).